zetaclient/chains/solana/observer/inbound.go

```diff
@@ -281,7 +288,28 @@ func (ob *Observer) BuildInboundVoteMsgFromEvent(event *clienttypes.InboundEvent
 		0,
 		event.CoinType,
 		event.Asset,
-		ob.ZetacoreClient().GetKeys().GetOperatorAddress().String(),
 		0, // not a smart contract call
+		crosschaintypes.ProtocolContractVersion_V1,
+		false, // not relevant for v1
 	)
 }
+
+// IsEventProcessable checks if the inbound event is processable
+func (ob *Observer) IsEventProcessable(event clienttypes.InboundEvent) bool {
```

Comment:
Let's unexport this

Reply:
It is exported to the unit tests. We have been following a pattern that separates unit test and source code in different packages, for example, `observer/observer_test`, `keeper/keeper_test`. Can we leave it exported in this PR until some day we can make a final choice on which pattern to follow?

Reply:
My point is that breaking long process into intermediate steps also have advantages. It could make source code easier to unit test such that we can examine it piece by piece instead of the whole piece like an e2e test.